source/dubregistry/web.d

```diff
@@ -205,10 +212,23 @@ class DubRegistryWebFrontend {
 		}
 
 		info.versions.sort!((a, b) => vcmp(a.id, b.id))();
+		return info;
+	}
 
+	void getDownload()
+	{
+		auto info = downloadInfo();
 		render!("download.dt", info);
 	}
 
+	@path("/download/LATEST")
+	void getLatest(HTTPServerResponse res)
+	{
+		auto info = downloadInfo();
+		enforceHTTP(!info.latest.empty, HTTPStatus.notFound, "No version available.");
```

Comment:
It's pretty long though :).
https://github.com/rejectedsoftware/vibe.d/pull/980